source/api/annex/notes/scheduled-changes.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Scheduled Changes"
+layout: spec
+tags: [annex, presentation-api, image-api]
+---
+
+# Introduction
+
+This document records scheduled changes that are intended to be made in the future, but must wait for new major releases as they're backwards incompatible.
+
+# Image API
+
+* Require support for the `square` region at compliance level 1.
+* Clarify that height and width are for calculating aspect ratio only, as there is no 'source' image.
+
```

Comment:
I considered it.  Transparency is good, getting to be part of the process is good ... but not sure if this is too much sausage making? We can put them in, and take them out if it's too weird ... or we can put them in with a subsequent PR.